src/main/java/uk/ac/ebi/subs/repository/repos/fileupload/FileRepository.java

```diff
@@ -48,6 +49,9 @@
     @PreAuthorizeSubmissionIdTeamName
     Page<File> findBySubmissionId(@P("submissionId") @Param("submissionId") String submissionId, Pageable pageable);
 
+    Stream<File> findBySubmissionIdAndStatusNot(
```

Comment:
Do we need a stream here, or would a count be more useful?

Reply:
Of course a count is more useful and I wanted to change it to a `countBy`, but for some reason - I blame the broken github service - I forgot to change this. I will do it now.